### ISSUE TITLE
jamonapi source code quality with findbugs check.

### ISSUE THREAD

**Issue by bwzhang2011** [stevensouza/jamonapi#10]
@stevensouza, when I compiled the jamonapi  with findbugs check, I found there're some built-in check errors found according to findbugs, so any idea on improve the source code quality as findbugs pointed out. I think it's a good idea with some small modification and improvement with respected to the release of 2.8.2

**Comment by stevensouza** [comment]
Yes. That is a good idea.

**Comment by bwzhang2011** [comment]
@stevensouza, any update with such issue ?

**Comment by stevensouza** [comment]
Did you use findbugs via a maven build?  Is it built into your build process?

**Comment by bwzhang2011** [comment]
@stevensouza, yes. please take a look at findbugs-maven-plugin. and you could find the resource at 
http://repo2.maven.org/maven2/org/codehaus/mojo/findbugs-maven-plugin/
